rulesets/java/jpinpoint-rules.xml

```diff
@@ -2379,15 +2380,26 @@ if (LOG.isDebugEnabled()) { // good
                 </value>
             </property>
         </properties>
+        <example>
+            <![CDATA[
+@SuppressWarnings("pmd:UnconditionalCreatedLogArguments") // Reason for suppression: the rule doesn't recognixe record getters (yet), and those are not expensive
+public void process(ConsumerRecord consRecord) {
+    ConsumerEvent event = consRecord.event(); // record getter
+    log.info("Received event '{}'", event); // pmd:UnconditionalCreatedLogArguments suppressed
+}
+             ]]>
+        </example>
     </rule>
 
     <rule name="UsingSuppressWarningsHighRisk"
           class="net.sourceforge.pmd.lang.rule.xpath.XPathRule"
           language="java"
           message="Using SuppressWarnings for a rule that is meant to prevent high risk problems."
           externalInfoUrl="https://github.com/jborgers/PMD-jPinpoint-rules/tree/pmd7/docs/JavaCodePerformance.md">
-        <description>(Informative - are you sure?) Problem: This rule detects high risk problems, suppressing them without full knowledge can lead to incidents like customer data mix-up, corrupt data, server crashes or very bad performance. &#13;
-            Solution: Suppress warnings judiciously based on full knowledge and report reasons to suppress (false positives) to the rule maintainers so the rule can be fixed. (jpinpoint-rules)</description>
+        <description>Problem: The suppressed rule detects high risk problems, suppressing the detected violations without full knowledge can lead to incidents like customer data mix-up, corrupt data, server crashes or very bad performance. &#13;
+            Solution: Suppress violations (warnings) judiciously based on full knowledge and report reasons to suppress (false positives) to the rule maintainers so the rule can be improved.
```

Comment:
use other word for judiciously, maybe better: "Solution: Only suppress violations (warnings) when you have complete understanding of the implications and valid reasons for doing so. When suppressing warnings, document your reasoning and report confirmed false positives to the rule maintainers to help improve rule accuracy."